model/src/main/scala/ch.epfl.scala.index.model/Project.scala

```diff
@@ -24,11 +28,18 @@ case class Project(
     organization: String,
     repository: String,
     github: Option[GithubInfo] = None,
-    keywords: List[String] = Nil,
+    keywords: Set[String] = Set(),
     defaultArtifact: Option[String] = None,
-    customScalaDocUrl: Option[String] = None,
+    artifacts: List[String] = Nil,
+    releaseCount: Int = 0,
+    customScalaDoc: Option[String] = None,
     documentationLinks: Map[String, String] = Map(),
+    deprecated: Boolean = false,
+    contributorsWanted: Boolean = false,
+    artifactDeprecations: Set[String] = Set(),
     logoImageUrl: Option[Url] = None,
+    background: Option[String] = None,
```

Comment:
@heathermiller this is actually pretty cool it let project owner customize their page.